tool/get-deprecations.ts

```diff
@@ -0,0 +1,67 @@
+// Generates the list of deprecations from spec/deprecations.yaml in the
+// language repo.
+
+import * as fs from 'fs';
+import {parse} from 'yaml';
+
+interface YamlData {
+  [key: string]: {
+    description: string;
+    'dart-sass': {
+      status: 'active' | 'future' | 'obsolete';
+      deprecated?: string;
+      obsolete?: string;
+    };
+  };
+}
+
+const yamlFile = 'build/sass/spec/deprecations.yaml';
+
+/**
+ * Converts a version string in the form X.Y.Z to be code calling the Version
+ * constructor, or null if the string is undefined.
+ */
+function toVersionCode(version: string | undefined): string | null {
+  if (!version) return null;
```

Comment:
Since this is only ever used as a string value, I think it should return the string `'null'` here.

Reply:
Done